utils/build-script-impl

```diff
@@ -1240,6 +1240,18 @@ if [[ ! "${SKIP_BUILD_SWIFTPM}" ]] ; then
      PRODUCTS=("${PRODUCTS[@]}" swiftpm)
 fi
 
+# Checks if a given product is enabled (i.e. part of $PRODUCS array)
+function contains_product() {
```

Comment:
This is called from a lengthy, complex switch statement, would prefer to keep this (unrelated) logic out of that.